ChangeLog

```diff
@@ -53,6 +54,8 @@ Changes
      underlying OS actually guarantees.
    * Fix configuration queries in ssl-opt.h. #2030
    * Ensure that ssl-opt.h can be run in OS X. #2029
+   * Remove MBEDTLS_MEMORY_BUFFER_ALLOC_C from 'full' configuration
```

Comment:
I don't think we need a changelog entry for that. What goes into `config.pl full` is an internal matter.

Reply:
I don't mind either way; removing.